packages/amplify_authenticator/lib/src/widgets/authenticator_banner.dart

```diff
@@ -27,7 +27,8 @@ MaterialBanner createMaterialBanner(
   required List<Widget> actions,
   required bool useAmplifyTheme,
 }) {
-  final bool isDark = AmplifyTheme.of(context).isDark;
```

Comment:
This was throwing because AmplifyTheme.of() calls InheritedConfig.of(), and InheritedConfig is a defendant of the widget where these subscriptions are set up. `Theme.of(context).brightness == Brightness.dark` is the same thing as `AmplifyTheme.of(context).isDark`.